src/Symfony/Component/Profiler/README.md

```diff
@@ -0,0 +1,45 @@
+Profiler Component
+==================
+
+Profiler collects information about each request made to your
+HttpKernel-based application and store them for later analysis.
```

Comment:
Problem at the moment is the ProfileListener, I moved it from the HttpKernel to prevent BC and not needing to rename it.